accounting/docs.md

```diff
@@ -86,6 +86,27 @@ A fee entry represents the on chain fees we paid for a transaction.
 - Reference: TransactionID:-1. 
 - Note: Note set for fees. 
 
+### Sweep
+A sweep is an on chain transaction which is used to sweep our own funds back to our wallet. This is required when 
```

Comment:
I think you may have accidentally deleted the last part of the sentence after "when".